traffic_ops/testing/api_contract/v4/requirements.txt

```diff
@@ -0,0 +1,8 @@
+Apache-TrafficControl==3.1.0
+attrs==22.2.0
+munch==2.5.0
+packaging==23.0
+pytest==7.2.1
+requests==2.28.2
+trafficcontrol==1.0.0
+urllib3==1.26.14
```

Comment:
what is `urllib3` used for? I don't see it being imported anywhere

Reply:
Urllib is used in conftest.py to parse Traffic Ops url. to get hostname and api_version.

Reply:
Right, it uses `urllib`, which is a part of the Python standard library, not `urllib3`

Reply:
Sure, Removed it